internal/dag/dag.go

```diff
@@ -571,6 +574,24 @@ type HeaderValueMatchDescriptorEntry struct {
 	Value       string
 }
 
+type VhRateLimitsType int
+
+const (
+	// VhRateLimitsOverride (Default) will use the virtual host rate limits unless the route has a rate limit policy.
+	VhRateLimitsOverride VhRateLimitsType = iota
+
+	// VhRateLimitsInclude will use the virtual host rate limits even if the route has a rate limit policy.
+	VhRateLimitsInclude
```

Comment:
Do we need `VhRateLimitsInclude` for this case?

Reply:
I don't have a strong opinion here. 
I kept them just in case someone wants to extend the implementation and supports another mode.